libs/platform/src/lib/components/form/radio/radio.component.html

```diff
@@ -0,0 +1,22 @@
+<ng-template #renderer>
+    <ng-container *ngTemplateOutlet="radio"></ng-container>
```

Comment:
I have commented this before on slack, this is not addressed.
1. I was asking, why do you need this ng-template `radio` and extra ng-container?